### ISSUE TITLE
Double sections

### ISSUE THREAD

**Issue by seagle0128** [alphapapa/magit-todos#127]
While pressing `g` twice, two Todos sections are displayed in magit-status buffer as below.

<img width="166" alt="image" src="https://user-images.githubusercontent.com/140797/162585451-321e57b3-0268-4b53-a87d-eb47cf389379.png">

**Comment by alphapapa** [comment]
What happens if you press `g` a third time?

**Comment by seagle0128** [comment]
> What happens if you press `g` a third time?

Only one TODOs section is displayed.

To be clear, I mean the issue appears while double pressing `g` quickly. Otherwise the behavior is normal.

**Comment by alphapapa** [comment]
Yes, there is nothing to stop the async process from being run multiple times at once and thereby inserting multiple sections.  AFAIK it does no harm and would probably not be worth writing code to prevent this.

**Comment by seagle0128** [comment]
I agree it does no hard, but I'd like suggest to check before inserting the TODO section if possible.

**Comment by alphapapa** [comment]
It could be done, but it's not a priority for me, because this falls into the "well, don't do that" category.  :)  Patches welcome.

**Comment by alphapapa** [comment]
See also: #145
